media/src/components/UserHeader.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+    Avatar,
+    Box,
+    Flex,
+    VStack,
+    Text,
+    MenuButton,
+    Menu,
+    Button,
+    Portal,
+    MenuList,
+    MenuItem,
+    useToast,
+    useColorMode
+  } from "@chakra-ui/react";
+  import { faker } from "@faker-js/faker";
+  import { Link } from "react-router-dom";
+  import { BsTwitter } from "react-icons/bs";
+  import { CgMoreO } from "react-icons/cg";
+  
+  const UserHeader = () => {
+    const toast = useToast();
+    const {colorMode} = useColorMode()
+  
+    const copyURL = async () => {
+      const currentURL = window.location.href;
+      navigator.clipboard.writeText(currentURL).then(() => {
+        toast({
+          title: "Account created.",
+          description: "Profile link copied",
+          status: "success",
+          duration: 3000,
+          isClosable: true,
+        });
+        console.log("http://localhost:5173/username");
+      });
```

Comment:
we should always have the error state handled. even if it's just a console.log or even an empty function. in this way, if there's an error, even if we don't care about it happening, at least it would not bring down the app. 

this would possibly be different if you're intention is to bubble the error up to an [ErrorBoundary](https://react.dev/reference/react/useTransition#displaying-an-error-to-users-with-error-boundary). But normally I use those as a last ditch catch all and would prefer each component to handle it's own error state.